tests/Exception/RequestExceptionTest.php

```diff
@@ -174,4 +175,30 @@ public function testObfuscateUrlWithUsernameAndPassword()
         $e = RequestException::create($r, new Response(500));
         $this->assertContains('http://user:***@www.oo.com', $e->getMessage());
     }
+
+    public function testGetResponseBodySummaryOfNonReadableStream()
+    {
+        $stream = new ReadSeekOnlyStream();
+        $stream->detach();
```

Comment:
I wonder why the stream is needed to be detached first.

Reply:
most likely a leftover from the previous test version, good catch :+1: